### PR TITLE
Patch releases for server

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description
Patch releases for server v3.4.6, v4.1.1 and v4.0.3

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/server/overview/release-notes.adoc [hunk 1]
```diff
@@ -11,7 +11,7 @@ version:
 :toc: macro
 :toc-title:
 
-Server v4.x offers the following:
+Server v4.1 offers the following:
```
Comment (rosieyohannan) [inline jekyll/_cci2/server/overview/release-notes.adoc:14]:
```suggestion
Server v4.0 offers the following:
```

File: jekyll/_cci2/server/v4.1/installation/migrate-from-server-3-to-server-4.adoc [hunk 2]
```diff
@@ -93,10 +93,10 @@ helm registry login cciserver.azurecr.io --username <image-registry-username>
 The Helm Diff tool is used to verify that the changes between your current install and the upgrade are expected.
 
 # diff command
-helm diff upgrade <release-name> -n <namespace> -f <path>/kots-exporter/output/helm-values.yaml --show-secrets --context 5 oci://cciserver.azurecr.io/circleci-server --version 4.0.2
+helm diff upgrade <release-name> -n <namespace> -f <path>/kots-exporter/output/helm-values.yaml --show-secrets --context 5 oci://cciserver.azurecr.io/circleci-server --version 4.0.3
```
Comment (rosieyohannan) [inline jekyll/_cci2/server/v4.1/installation/migrate-from-server-3-to-server-4.adoc:96]:
```suggestion
helm diff upgrade <release-name> -n <namespace> -f <path>/kots-exporter/output/helm-values.yaml --show-secrets --context 5 oci://cciserver.azurecr.io/circleci-server --version 4.1.1
```

File: jekyll/_cci2/server/v4.1/installation/migrate-from-server-3-to-server-4.adoc [hunk 3]
```diff
@@ -141,7 +141,7 @@ Next, execute the helm-diff command and review the output.
 ----
 helm registry login cciserver.azurecr.io -u <image-registry-username>
 
-helm diff upgrade <release-name> -n <namespace> -f <path>/kots-exporter/output/helm-values.yaml --show-secrets --context 5 oci://cciserver.azurecr.io/circleci-server --version 4.0.2
+helm diff upgrade <release-name> -n <namespace> -f <path>/kots-exporter/output/helm-values.yaml --show-secrets --context 5 oci://cciserver.azurecr.io/circleci-server --version 4.0.3
```
Comment (rosieyohannan) [inline jekyll/_cci2/server/v4.1/installation/migrate-from-server-3-to-server-4.adoc:144]:
```suggestion
helm diff upgrade <release-name> -n <namespace> -f <path>/kots-exporter/output/helm-values.yaml --show-secrets --context 5 oci://cciserver.azurecr.io/circleci-server --version 4.1.1
```

File: jekyll/_cci2/server/v4.1/installation/migrate-from-server-3-to-server-4.adoc [hunk 4]
```diff
@@ -175,7 +175,7 @@ Once your helm-value file is verified, run the following commands to upgrade the
 Our helm registry is stored in an azure private registry. You will be provided a username and token to access the registry.
 [source,shell]
 ----
-helm upgrade circleci-server -n <namespace> -f <path>/kots-exporter/output/helm-values.yaml oci://cciserver.azurecr.io/circleci-server --version 4.0.2 --force
+helm upgrade circleci-server -n <namespace> -f <path>/kots-exporter/output/helm-values.yaml oci://cciserver.azurecr.io/circleci-server --version 4.0.3 --force
```
Comment (rosieyohannan) [inline jekyll/_cci2/server/v4.1/installation/migrate-from-server-3-to-server-4.adoc:178]:
```suggestion
helm upgrade circleci-server -n <namespace> -f <path>/kots-exporter/output/helm-values.yaml oci://cciserver.azurecr.io/circleci-server --version 4.1.1 --force
```

**Review by ryanwohara — APPROVED** [review]

**Review by kbatuigas — APPROVED** [review]
LGTM 🙏🏾

**Pull request merged**